core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -205,7 +205,7 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
         <md-card ng-repeat="exploration in explorationsList | orderBy:sortByFunction:isCurrentSortDescending track by exploration.id"
                  class="oppia-activity-summary-tile oppia-dashboard-card-view-item protractor-test-exploration-dashboard-card"
                  is-private="exploration.status === 'private'">
-          <a ng-href="/create/<[exploration.id]>">
+          <a ng-href="/create/<[exploration.id]>#/gui/Introduction">
```

Comment:
Note that "Introduction" just happens to be the name of one of the states in the default exploration. I don't think you want this to be hardcoded.

Also, starting from being logged out, then clicking the 'create' button from, e.g., the splash page, then clicking the back button, causes a second new exploration to be immediately created. I don't think this is what you want.